copy-this-template-directory/index.md

```diff
@@ -1,16 +1,924 @@
+
+Click `Triggers`, then `Add Trigger`
+
+***
+
+![GAS 10](/copy-this-template-directory/GAS_10.png "GAS 10")
+
+Select function triggerResubscribeOnceWeek
+
+Deployment - Head
+
+Select event source - Time-driven
+
+Select type - Week Timer
+
+Select day of week and time - your choice
+
+Failure notifications - Notify me immediately
+
+Click `Save`
+
+&nbsp; 
+
+### Script source code - list of functions
+
+The source code of the script is organized into several functions (but there is a lot of space for refactoring and improvements)
+
+`writelog` - utility function to write to the spreadsheet log entry
+
+`listCalendars` - function for manual launch - to list available for user calendars info (IDs, description)
+
+`triggerResubscribeOnceWeek` - function for once-a-week time trigger. Will resubscribe to calendar push notifications (1 week is the maximum allowed time)
+
+`runManual_processSyncedEvents` - function for manual launch - get events stream and process it (send to Directus)
+
+`runManual_getSyncedEvents` - function for manual launch - set initial sync token, nothing sent to Directus
+
+`getConfig` - utility function for retrieving Config set in spreadsheet
+
+`updateConfig` - utility function for updating certain Config in the spreadsheet
+
+`callDirectusWebhook` - actual data sent to Directus
+
+`processSyncedEvents` - get events stream and process it (send to Directus)
+
+`doPost` - main function, special name doPost means that when the web app is published, POST requests are processed here
+
+`callCalendarEventsWatch_stop` - utility function for stopping push notifications
+
+`callCalendarEventsWatch` - utility function for registering URL for receiving push notifications
+
+`getSyncedEvents` - utility function for getting a list of new / updated / deleted events in Google Calendar, using special "syncToken"
+
+&nbsp; 
+
+#### Script update deployment
+
+The deployed web app has versions, so if you update doPost function (or something called from it) and want these changes to have an effect - a new version of the web app needs to be deployed.
+
+![GAS 07](/copy-this-template-directory/GAS_07.png "GAS 07")
+
+Manage Deployments → `Edit` (pencil icon button)
+
+![GAS 08](/copy-this-template-directory/GAS_08.png "GAS 08")
+
+Choose `New version`, then click `Deploy`.
+
+The URL of the web app should stay the same (if you updated the current deployment and didn't create new deployment)
+
+***
+
+&nbsp; 
+
+### Ready to roll
+
+I hope you are not exhausted after reading this post and repeating all the actions required for this project to run.
+
+Everything is set to fully automatic two-way sync. You can try it by adding items in Directus and adding events in the Calendar.
 
-<!-- ## Your Sections Here -->
+You can check logs in the log panel for each of  Directus Flows. And you can check spreadsheet `log` sheet.
 
-## Summary
+Feel free to send me a message if you know how to improve this project.
```

Comment:
Actually write a summary here which outlines again the steps we took at a high-level.